internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -80,7 +105,7 @@ func (df *Dockerfile) GetExposedPorts() ([]uint16, error) {
 	return ports, err
 }
 
-// parse takes a Dockerfile and fills in struct members based on methods like parseExpose and (TODO) parseHealthcheck
+// parse takes a Dockerfile and fills in struct members based on methods like parseExpose and parseHealthcheck
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
> Comments should begin with the name of the thing being described and end in a period

This wiki page is pretty neat and contains all sorts of best-practices, if you'd like to take a look when you've time :)